ui/src/shared/components/time_series/TimeSeries.tsx

```diff
@@ -228,8 +237,8 @@ class TimeSeries extends Component<Props, State> {
 
   private get isFluxSource(): boolean {
     // TODO: update when flux not separate service
-    const {service} = this.props
-    return !!service
+    const {queries} = this.props
+    return queries[0].type === QueryType.Flux
```

Comment:
could you use a get or getdeep here so it doesn't error in the case that there are no queries

Reply:
will do!